src/content/home.mdx

```diff
@@ -0,0 +1,12 @@
+---
+layout: page
+title: 'Home'
+---
+
+<ContentSection>
+
+### We're shaping the future of digital journalism. <br/> [Join us](https://workforus.theguardian.com)
```

Comment:
Markdown in components work!

Reply:
Interesting. Do we know how hard it is to style these? There's the ability to swap out default elements like `h1`, `h2`, etc., for custom elements that can be styled how we wish. 
https://www.gatsbyjs.com/docs/mdx/customizing-components/

Maybe this approach will be compatible with the markdown, meaning that we can set default styles for some of these?